### ISSUE TITLE
Admin Inline show_change_link = True does nothing

### ISSUE THREAD

**Issue by nyejon** [viewflow/django-material#312]
Hi,

In the inline admin forms, the show_change_link does not appear when the option is set to 'True'.

From the django docs:

> InlineModelAdmin.show_change_link¶
Specifies whether or not inline objects that can be changed in the admin have a link to the change form. Defaults to False.

**Comment by Sahana312** [comment]
@nyejon  - Did you find any workaround for this? As I need the same thing to be done in my project.

**Comment by nyejon** [comment]
Not yet, I've just left it showing the details only for now.

**Comment by opw0011** [comment]
I have the same problem. Here is a tmp solution I found on stack overflow:
```py
from django.utils.html import format_html
from django.core.urlresolvers import reverse

class MyModelInline(admin.TabularInline):
    model = MyModel

    def admin_link(self, instance):
        url = reverse('admin:%s_%s_change' % (instance._meta.app_label,  
                                              instance._meta.model_name),
                      args=(instance.id,))
        return format_html(u'<a href="{}">Edit</a>', url)
        # … or if you want to include other fields:
        return format_html(u'<a href="{}">Edit: {}</a>', url, instance.title)

    readonly_fields = ('admin_link',)
```
https://stackoverflow.com/a/15055057

**Comment by angelchikage** [comment]
Under admin\templates\material\admin\edit_inline
Both tabular.html and stacked.html has no code about **inline_admin_form.model_admin.show_change_link** to add <a href = ... >

**Comment by mmzepedab** [comment]
Has anyone come with a solution for this? It would be nice if it gets implemented.

**Comment by kmmbvnr** [comment]
Closed due #373
